internal/coreinternal/attraction/attraction.go

```diff
@@ -84,8 +84,8 @@ type ActionKeyValue struct {
 	//           Either Value, FromAttribute or FromContext must be set.
 	// DELETE  - Deletes the attribute. If the key doesn't exist,
 	//           no action is performed.
-	// HASH    - Calculates the SHA-1 hash of an existing value and overwrites the
-	//           value with it's SHA-1 hash result.
+	// HASH    - Calculates the SHA-256 hash of an existing value and overwrites the
+	//           value with it's SHA-256 hash result.
```

Comment:
You could make the call that this is a breaking change. If so, I think we can maybe offer a second HASH function such as HASH-SHA2-256.